namespaces/collectd.yml

```diff
@@ -326,18 +426,9 @@ namespace:
     description: >
       Corresponds to collectd's df plugin.
     fields:
-    - name: space_type
```

Comment:
The "space type" is in the "type_instance" and also in the value field nested name.
mounted_volume - same as I commented before, this is plugin specific and I think we should leave as id in plugine_instance.